src/main/java/org/sonarsource/scanner/maven/bootstrap/MavenProjectConverter.java

```diff
@@ -122,6 +123,18 @@ public class MavenProjectConverter {
 
   private final MavenCompilerResolver mavenCompilerResolver;
 
+  public Set<Path> getSkippedBasedDirs() {
+    return skippedBasedDirs;
+  }
+
+  private final Set<Path> skippedBasedDirs = new HashSet<>();
+
+  private boolean sourceDirsIsOverridden = false;
+
+  public boolean isSourceDirsIsOverridden() {
```

Comment:
Rename to be less redundant
```suggestion
  public boolean isSourceDirsOverridden() {
```